Add step in VNet setup (#3689)
astronomer/docs: astro/connect-azure.md

    ```sh
    # add Astronomer Service Principal
    az ad sp create --id a67e6057-7138-4f78-bbaf-fd9db7b8aab0
    
    # Grant sets of permissions to the Astronomer Service Principal
    az role assignment create \
      --assignee a67e6057-7138-4f78-bbaf-fd9db7b8aab0 \
      --role "Network Contributor" \
      --scope <vnet-id>
    
    az role assignment create \
      --assignee a67e6057-7138-4f78-bbaf-fd9db7b8aab0 \
      --role "Network Contributor" \
      --scope <resource-group-id>
    
    az role assignment create \
      --assignee a67e6057-7138-4f78-bbaf-fd9db7b8aab0 \
      --role "Network Contributor" \
      --scope <subscription-id>
    ```

3. Contact [Astronomer support](https://cloud.astronomer.io/open-support-request) to tell them that you have granted permissions to the Astronomer Service Principal. In addition, provide the following details in your request:

    - Astro cluster ID and name
    - Azure TenantID and Subscription ID of the target VNet
    - Region of the target VNet
    - VNet ID of the target VNet
    - Classless Inter-Domain Routing (CIDR) block of the target VNet

> After receiving your request, Astronomer support initiates a peering request and creates the routing table entries in the Astro VNet. To allow multidirectional traffic between Airflow and your organization's data sources, the owner of the target VNet needs to accept the peering request and create the routing table entries in the target VNet.

Author: How/ when does Astronomer delete the permissions that were just created? @afastronomer

Reviewer: Once VNet peering connectivity is established and confirmed by CRE, customer can delete those role assignments.

> After receiving your request, Astronomer support initiates a peering request and creates the routing table entries in the Astro VNet. To allow multidirectional traffic between Airflow and your organization's data sources, the owner of the target VNet needs to accept the peering request and create the routing table entries in the target VNet.

Reviewer: This part is applicable for AWS but Azure. It has to be something like:
```
After receiving your request, Astronomer support creates a VNet peering connectivity from both sides. No other actions are needed from your side. Astronomer support will let you know when connectivity is ready for use.
```

> After receiving your request, Astronomer support initiates a peering request and creates the routing table entries in the Astro VNet. To allow multidirectional traffic between Airflow and your organization's data sources, the owner of the target VNet needs to accept the peering request and create the routing table entries in the target VNet.

Author: ```suggestion
After receiving your request, Astronomer support initiates a peering request and creates the routing table entries in the Astro VNet. To allow multidirectional traffic between Airflow and your organization's data sources, the owner of the target VNet needs to accept the peering request and create the routing table entries in the target VNet.

When the network connection is confirmed, you can delete the temporary roles you created using the [Azure CLI](https://learn.microsoft.com/en-us/azure/role-based-access-control/role-assignments-remove#azure-cli).
```

1. Retrieve the following information from the target Azure cluster that you want to connect with:

    - Azure Subscription ID.
    - VNet ID.
    - Resource Group ID.

2. Run the following Azure CLI commands to give Astronomer support temporary permissions to establish a VNet peering connection:

    ```sh
    # add Astronomer Service Principal
    az ad sp create --id a67e6057-7138-4f78-bbaf-fd9db7b8aab0
    
    # Grant sets of permissions to the Astronomer Service Principal
    az role assignment create \
      --assignee a67e6057-7138-4f78-bbaf-fd9db7b8aab0 \
      --role "Network Contributor" \
      --scope <vnet-id>
    
    az role assignment create \
      --assignee a67e6057-7138-4f78-bbaf-fd9db7b8aab0 \
      --role "Network Contributor" \
      --scope <resource-group-id>
    
    az role assignment create \
      --assignee a67e6057-7138-4f78-bbaf-fd9db7b8aab0 \
      --role "Network Contributor" \
      --scope <subscription-id>
    ```

3. Contact [Astronomer support](https://cloud.astronomer.io/open-support-request) to tell them that you have granted permissions to the Astronomer Service Principal. In addition, provide the following details in your request:

>     - Astro cluster ID 
>     - Azure TenantID and Subscription ID of the target VNet
>     - VNet ID of the target VNet

Reviewer 2: ```suggestion
    - Astro Cluster ID 
    - Azure Tenant ID and Subscription ID of the target VNet
    - VNet ID of the target VNet
```

1. Retrieve the following information from the target Azure cluster that you want to connect with:

    - Azure Subscription ID.
    - VNet ID.
    - Resource Group ID.

2. Run the following Azure CLI commands to give Astronomer support temporary permissions to establish a VNet peering connection:

    ```sh
    # add Astronomer Service Principal
    az ad sp create --id a67e6057-7138-4f78-bbaf-fd9db7b8aab0
    
    # Grant sets of permissions to the Astronomer Service Principal
    az role assignment create \
      --assignee a67e6057-7138-4f78-bbaf-fd9db7b8aab0 \
      --role "Network Contributor" \
      --scope <vnet-id>
    
    az role assignment create \
      --assignee a67e6057-7138-4f78-bbaf-fd9db7b8aab0 \
      --role "Network Contributor" \
      --scope <resource-group-id>
    
    az role assignment create \
      --assignee a67e6057-7138-4f78-bbaf-fd9db7b8aab0 \
      --role "Network Contributor" \
      --scope <subscription-id>
    ```

3. Contact [Astronomer support](https://cloud.astronomer.io/open-support-request) to tell them that you have granted permissions to the Astronomer Service Principal. In addition, provide the following details in your request:

> 

Reviewer: We do not need cluster name, ID is the most suitable information.
We do not need region and CIDR block.

>     - Astro cluster ID and name
>     - Azure TenantID and Subscription ID of the target VNet
>     - Region of the target VNet
>     - VNet ID of the target VNet
>     - Classless Inter-Domain Routing (CIDR) block of the target VNet

Reviewer 2: ```suggestion
    - Astro cluster ID 
    - Azure TenantID and Subscription ID of the target VNet
    - VNet ID of the target VNet
```

> After receiving your request, Astronomer support creates a VNet peering connection from both sides. No other actions are needed from your side. Astronomer support will let you know when the connection is ready to use.

Author: ```suggestion
After receiving your request, Astronomer support creates a VNet peering connection between the two VNets. No other actions are required from you. Astronomer support will notify you when the connection is ready to use.
```
